migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -129,8 +131,12 @@ For more information, see xref:../storage/understanding-persistent-storage.adoc#
 
 * Amazon Web Services (AWS) Elastic Block Storage (EBS)
 * Azure Disk
+* Azure File
 * Google Cloud Platform Persistent Disk (GCP PD)
 * OpenStack Cinder
+* VMWare vSphere
+
+Please note that you can opt into VMware vSphere. However, as of {product-title} {product-version} is not available by default.
```

Comment:
`However, as of {product-title} {product-version} is not available by default.`

I couldn't get 'what' is not available by default. The sentence seems to miss subject. Let me know if I am misunderstanding something here.